python/ml4ir/base/tests/test_stat_analysis_calculation.py

```diff
@@ -1,17 +1,18 @@
 import numpy as np
+import pandas as pd
 from scipy import stats
 import unittest
 import warnings
-from ml4ir.applications.ranking.t_test import perform_click_rank_dist_paired_t_test, compute_stats_from_stream
+from ml4ir.applications.ranking.t_test import perform_click_rank_dist_paired_t_test, compute_stats_from_stream, compute_org_running_variance_for_metrics, StreamVariance
 
 warnings.filterwarnings("ignore")
 
 np.random.seed(123)
 
 
-class TestTtestCalculation(unittest.TestCase):
+class StatisticalAnalysisCalculation(unittest.TestCase):
```

Comment:
[Minor] @mohazahran Shouldn't this start with `Test` to be picked up? Regardless, might be good to add the prefix for consistency.